packages/sanity/src/core/studio/components/navbar/search/components/searchResults/SearchResults.tsx

```diff
@@ -49,12 +49,18 @@ export function SearchResults({disableIntentLink, inputElement, onItemSelect}: S
   /**
    * Add current search to recent searches, trigger child item click and close search
    */
-  const handleSearchResultClick = useCallback(() => {
-    if (recentSearchesStore) {
-      recentSearchesStore.addSearch(terms, filters)
-    }
-    onClose?.()
-  }, [filters, onClose, recentSearchesStore, terms])
+  const handleSearchResultClick = useCallback(
+    (e: MouseEvent<HTMLElement>) => {
+      if (recentSearchesStore) {
+        recentSearchesStore.addSearch(terms, filters)
+      }
+      // if the cmd key is pressed, we don't want to close the search
```

Comment:
I think the comment should say _why_ we dont want to close it (its to support open result in new link)